plugin/evm/atomic/sync/atomic_sync_parser.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package sync
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/coreth/plugin/evm/message"
+	"github.com/ava-labs/libevm/crypto"
+)
+
+var _ message.SyncableParser = (*atomicSyncSummaryParser)(nil)
+
+type atomicSyncSummaryParser struct{}
+
+func NewAtomicSyncSummaryParser() *atomicSyncSummaryParser {
+	return &atomicSyncSummaryParser{}
+}
+
+func (a *atomicSyncSummaryParser) ParseFromBytes(summaryBytes []byte, acceptImpl message.AcceptImplFn) (message.Syncable, error) {
```

Comment:
Do we ever parse something _not_ from bytes?
```suggestion
func (a *atomicSyncSummaryParser) Parse(summaryBytes []byte, acceptImpl message.AcceptImplFn) (message.Syncable, error) {
```